src/AddTodoForm.jsx

```diff
@@ -0,0 +1,13 @@
+const AddTodoForm = () => {
+    return (
+        <>
+            <form action="">
+                <label id="todoTitle" title="Title"></label>
```

Comment:
See https://react.dev/reference/react-dom/components/input#providing-a-label-for-an-input for how to attach a label to an input element. We can't give two different elements (the label and the input) the same `id` property, that must be unique to exactly one element. Let's either nest the text and the input inside the `form` element or use the `htmlFor` attribute on the label (see the link for help on both).